syntax = "proto3";

package tateyama.proto.test;

option java_package = "com.nautilus_technologies.tateyama.proto";
option java_outer_classname = "TestProtos";

// common test msg
message TestMsg {
  // the request message version.
  uint64 message_version = 1;

  // reserved for system use
  reserved 2 to 10;

  // the destination service ID.
  uint64 id = 11;

}